3-UsingAPIs/Week1/homework/ex4-pokerDiceAll.js

```diff
@@ -18,10 +18,11 @@ const rollDice = require('../../helpers/pokerDiceRoller');
 function rollTheDices() {
   // TODO Refactor this function
   const dices = [1, 2, 3, 4, 5];
-  const myAllPromises = dices.map((element) => rollDice(element));
-  Promise.all(myAllPromises)
-    
+  const combinedPromises = dices.map((element) => rollDice(element));
+
+  return Promise.all(combinedPromises);
```

Comment:
There was no need to change the name of the promises array `myAllPromises` to `combinedPromises`.  Or was it a copy and paste job from the work of Cemal (including blank line 22)?

Look here: https://github.com/CmlSph/Homework/blob/c3255b78bbee1b09712547d4e5a14720912abf7e/3-UsingAPIs/Week1/homework/ex4-pokerDiceAll.js#L18

Just a reminder from the HackYourFuture Code of Conduct:

>_Do not copy code. You want to develop your own coding skills and learn! If you copy code, you need to cite your source through a comment on your code with an explanation of the functionality of the copied code._